pkg/restapi/comparator/operation/openapi/authorization.go

```diff
@@ -0,0 +1,276 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
Looks like these models were generated from the wrong spec - should be from [this one](https://github.com/trustbloc/edge-service/blob/0539d17f92353cedb99687e5ad34369a6a4ff90f/cmd/comparator-rest/docs/openapi.yaml).

Reply:
done